### PR TITLE
Print warning on missing model.

### PR DESCRIPTION
**Pull request by tabergma**
**Proposed changes**:
- Print error if no NLU model for testing was found

**Status (please check what you already did)**:
- [x] made PR ready for code review
- [ ] added some tests for the functionality
- [ ] updated the documentation
- [ ] updated the changelog
- [x] reformat files using `black` (please check [Readme](https://github.com/RasaHQ/rasa_nlu#code-style) for instructions)

### CODE REVIEW

**Comment by tabergma** [comment]
@tmbo 👍 
Tests are failing due to:
`ERROR: No matching distribution found for rasa_core_sdk~=1.0.0rc2 (from -r requirements.txt (line 35))`
Was there an update?

File: rasa/test.py [hunk 1]
```diff
@@ -77,20 +77,31 @@ def test_core(
             rasa.core.test(stories, _agent, out_directory=output, **kwargs)
         )
     else:
-        logger.error(
-            "Not able to test. Make sure both models, core and " "nlu, are available."
+        print_error(
+            "Not able to test. Make sure both models, core and nlu, are available."
```
Comment (MetcalfeTom) [inline rasa/test.py]:
```suggestion
            "Not able to test. Make sure both models "
            "- core and nlu - are available."
```

**Comment by tmbo** [comment]
Yes, but build is still pending on travis...should be fixed in a second

**Comment by tmbo** [comment]
Ok sdk build succeeded, should be green now.

**Pull request merged**